extensions/clusters/clusters.go

```diff
@@ -1060,6 +1228,38 @@ func DeleteK3SRKE2Cluster(client *rancher.Client, clusterID string) error {
 	return nil
 }
 
+// UpdateRKE1Cluster is a "helper" functions that takes a rancher client, old rke1 cluster config, and the new rke1 cluster config as parameters.
+func UpdateRKE1Cluster(client *rancher.Client, cluster, updatedCluster *management.Cluster) (*management.Cluster, error) {
```

Comment:
I see that this was here for rke2/k3s already but IMO these helpers shouldn't live in clusters.go. We were intentionally not putting functions that `wait` in clusters.go, as that is part of verify.

Reply:
Heard. Personally, I think that's a task that's outside the scope of this. I would want to then move all of the functions that do that into verify, not just this new function.

Reply:
would you mind making a task for this?

Reply:
Yep, made over here: https://github.com/rancher/qa-tasks/issues/1431